recipes/text_extensions_for_pandas/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "text_extensions_for_pandas" %}
+{% set version = "0.1b2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CODAIT/text-extensions-for-pandas/archive/{{ version }}.tar.gz
+  sha256: ab14b12dcea4b6928c2e1e5c83e3d23dd2b14baee1954504460ed93dd321405c
+
+build:
+  number: 0
+  skip: true  # [win,py<37]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.17
+    - pandas >=1.0.3,<1.2.0
+    - pyarrow >=1.0.0
+    - regex
+    - memoized-property  # [py36,py37]
+    - requests
+
+test:
+  imports:
+    - text_extensions_for_pandas
+  requires:
+    - python
+    - pip
+    - pytest
+    - pytorch
+    - transformers >=3.0.0
+    - spacy ==2.3.2
+    - ibm-watson
+  files:
+    - test_data
```

Comment:
I imagine these are not in the tarball but if they are simple python tests, I mean no compiled extension is being tested here, you probably do not need them b/c they are run upstream.

It is not a problem to ship and test it here but shipping test data like that can cause a disconnection with the upstream source of the test data repo. I'm imagine those come from upstream package repo, right?

Reply:
True, they are being run upstream and I don't want to add test data in the conda package. I thought including this would just run the tests before creating the package, just as additional check, and not add the data to the final package? I'll remove this since it's not necessary

Reply:
Would you just recommend having the import check then?

Reply:
It won't add the data to the final package. But, while extra testing is always nice, you can easily get "disconnected" and diverge from the test data upstream when copying it manually like this. It is not an easy problem to solve. When tests are a must we take it like that but b/c this is a pure python pkg, no compiled extensions, it is ok to use only the import test.

Reply:
Sounds good. I have it set to run the import test only now, which should be fine.